azure-pipelines.yml

```diff
@@ -97,22 +97,13 @@ jobs:
   - checkout: self
     persistCredentials: true
 
-  - template: azure-pipelines-templates/install-nuget.yml
-
-  - task: DotNetCoreCLI@2
-    displayName: Restore NuGet packages
-    condition: succeeded()
-    inputs:
-      command: restore
-      verbosityRestore: minimal
-      projects: $(System.DefaultWorkingDirectory)/tools/VersionCop/VersionCop.sln
-      feedsToUse: config
-      nugetConfigPath: $(System.DefaultWorkingDirectory)/tools/VersionCop/NuGet.Config
-
-  - script: dotnet build VersionCop.sln -c 'Release' -p:Configuration=Release -p:Platform="Any CPU" -p:PublicRelease=true --no-restore /t:build,pack
+  - script: |
```

Comment:
Why do you think is is better to use a scrip approach rather than the DotNetCoreCLI task?
If we're changing this, then let's go with the DotNetCoreCLI  task all the way. 

Other comments: 
- you're calling it twice for build and pack, when this is done in a single strike with the existing version
- there's a need to set the PublicRelease property in order for nbgv to do it's thing properly

The restore task seems dumb and unnecessary, but apparently is not according to the docs. That's why it's implemented the way it is. See here:
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops